sefaria/model/text.py

```diff
@@ -2032,7 +2032,7 @@ def __init__(self, oref, lang="en", vtitle=None, exclude_copyrighted=False, actu
         self.is_merged = False
         self.sources = []
 
-        if self.lang not in self._oref.index_node.supported_languages:
+        if self.lang != self._oref.index_node.parent.lexicon.version_lang:
```

Comment:
`index_node` can be any `VirtualNode`, not just a `DictionaryEntryNode` (the only other example right now is `SheetNode`). I think instead we need a function on `VirtualNode` called `supports_language(lang)`. Then `DictionaryEntryNode` can use `parent.lexicon.version_lang` to make sure the languages match but `SheetNode` will always return `True`.